src/physics/em/detail/PhysicsConstants.hh

```diff
@@ -0,0 +1,58 @@
+//---------------------------------*-CUDA-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file PhysicsConstants
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Types.hh"
+#include "base/Macros.hh"
+#include "base/Constants.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Physical constants which are derived from fundamental constants.
+ *
+ * Derived constants    | Unit                  | Notes
+ * -------------------- | --------------------- | ------------
+ * electron_mass_c2()   | g * (m/s)^2           |
+ * migdal_constant()    | cm^3                  | Bremsstrahlung
+ * lpm_constant()       | MeV/cm                | Relativistic Bremsstrahlung
+ */
+
+//!@{
+//! Constant functions
+CELER_CONSTEXPR_FUNCTION real_type electron_mass_c2()
+{
+    using namespace constants;
+
+    return electron_mass * c_light * c_light;
+}
+
+CELER_CONSTEXPR_FUNCTION real_type migdal_constant()
+{
+    using namespace constants;
+    using namespace units;
+
+    return 4 * pi * r_electron * lambdabar_electron * lambdabar_electron;
+}
+
+CELER_CONSTEXPR_FUNCTION real_type lpm_constant()
```

Comment:
This return type is inconsistent with the units in your documentation I think? Should this be returning a quantity in `MevEnergy`? And if that's confusing (since this is energy per "native" length) you could define a new quantity in this file:
```
using MevPerCm  = Quantity<UnitDivide<units::Mev, units::NativeUnit>>;
```

Reply:
Okay.  Changed as suggested.